snuba_sdk/visitors.py

```diff
@@ -264,6 +264,95 @@ def _visit_totals(self, totals: bool) -> str:
         return f"{totals}"
 
 
+class TranslationMQL(ExpressionVisitor[str]):
+    def __init__(self, use_entity_aliases: bool = False):
+        self.use_entity_aliases = use_entity_aliases
+
+    def _stringify_scalar(self, value: ScalarType) -> str:
```

Comment:
Why did you change this?

Reply:
The `_stringify_scalar` in the `Translation` class does a few extra things that are SnQL specific. For example, it prepends `array` and `tuple` to certain scalars which we don't want for MQL.

Reply:
Fair enough.